multiaccounts/settings/columns.go

```diff
@@ -402,6 +402,21 @@ var (
 		dBColumnName:   "wallet_token_preferences_group_by_community",
 		valueHandler:   BoolHandler,
 	}
+	ShowCommunityAssetWhenSendingTokens = SettingField{
```

Comment:
@Cuteivist I guess we should add all 3 properties to sync protobuf, to have them synced among devices and when local pairing as well.